pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```diff
@@ -0,0 +1,45 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkgconfig, go, gnome3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+
+  nativeBuildInputs = [ pkgconfig go ];
+  
+  buildInputs = [ gnome3.glib gnome3.gtk libhandy ];
+
+  desktopFile = makeDesktopItem {
+		name = pname;
+    desktopName = "gtkcord";
+		exec = "gtkcord3";
```

Comment:
You appear to be using a mix of tabs and spaces for indentation. Please indent the file consistently using 2 spaces.